ext/AbstractDifferentiationChainRulesCoreExt.jl

```diff
@@ -0,0 +1,13 @@
+module AbstractDifferentiationChainRulesCoreExt
+
+import AbstractDifferentiation as AD
+using ChainRulesCore: ChainRulesCore
```

Comment:
This is not being imported using the `..ChainRulesCore` that's needed for Requires

Reply:
It's not loaded with Requires.